syntax = "proto3";

package logging;

service LoggingService {
  rpc LogMessage(LogRequest) returns (LogResponse);
  rpc GetMessages(Empty) returns (MessagesResponse);
}

message LogRequest {
  string id = 1;
  string msg = 2;
}

message LogResponse {
  bool success = 1;
  string error = 2;
}

message Empty {}

message MessagesResponse {
  repeated string messages = 1;
}
